utils/removeDiscordRoleFromUser.js

```diff
@@ -0,0 +1,44 @@
+const discordRolesModel = require("../models/discordactions");
+const discordServices = require("../services/discordService");
+const { logType } = require("../constants/logs");
+const { addLog } = require("../models/logs");
+
+/**
+ * Removes a Discord role from a user using Discord Id.
+ *
+ * @param {Object} userData - User data.
+ * @param {string} discordId - User's Discord ID.
+ * @param {string} roleid - Discord Role ID.
+ *
+ * @returns {Promise<Object>} - Result with success status and message.
+ */
+export const removeDiscordRoleFromUser = async (userData, discordId, roleid) => {
+  try {
+    const role = await discordRolesModel.isGroupRoleExists({ roleid });
+
+    if (!role.roleExists) {
+      throw new Error("Role doesn't exist");
+    }
+
+    const roleData = role.existingRoles.docs[0].data();
+
+    await discordServices.removeRoleFromUser(roleData.roleid, discordId, userData);
+
+    const { wasSuccess } = await discordRolesModel.removeMemberGroup(roleData.roleid, discordId);
+    if (!wasSuccess) {
+      throw new Error("Role deletion failed");
+    }
+
+    await addLog(
+      logType.REMOVE_UNVERIFIED_ROLE,
+      { roleid, userid: discordId },
+      { message: "Unverified role removed successfully" }
```

Comment:
If this is a generic function. Then why this message?

Reply:
{ message: "Unverified role removed successfully" } - this is the body that is getting stored in the logs collection. Seen in few other places that this type of example is being used. That's why

Reply:
I am saying if it is a generic function then why the message is `Unverified role removed successfully`?

Reply:
Okay got it. This was a bad mistake from my end. Please have a look to the updated change